.github/workflows/mysql_hardening.yml

```diff
@@ -79,6 +79,11 @@ jobs:
             sudo apparmor_parser -R /etc/apparmor.d/usr.sbin.mysqld
         if: ${{ startsWith(matrix.molecule_distro, 'Debian') }}
 
+      - name: Temporary fix for roles
```

Comment:
Can you add some docs to this, especially when we can remove this temporary fix?